### DISCUSSION TITLE
Bulk update or create devices in ansible playbook

### DISCUSSION THREAD

**Discussion by PauletteASuare** [netbox-community/ansible_modules#1063] [Q&A]
Hello,
Is it possible to bulk update or create devices in ansible playbook?

**Comment by sc68cal** [comment]
Using the NetBox ansible modules, you would usually use a `loop` construct to create/update multiple devices.

**Comment by peylight** [comment]
@sc68cal Is it faster if we use pynetbox batch update?

**Reply by sc68cal** [reply]
Yes it will be faster in execution time, but may take longer to write the code
